app/views/lessons/project_submissions/index.html.erb

```diff
@@ -17,7 +17,7 @@
           userSubmission: @user_submission
         }
       ) %>
-  <div class="text-center">
+  <div class="text-center" style="overflow-x: auto;">
```

Comment:
Nit: Can we use the tailwind class for this please?
```suggestion
  <div class="text-center overflow-x-auto">
```

Reply:
Yes, I'll do that right away. Thanks.